daprdocs/content/en/reference/cli/dapr-mtls--export.md

```diff
@@ -0,0 +1,29 @@
+---
+type: docs
+title: "mtls export CLI command reference"
+linkTitle: "mtls export"
+description: "Detailed information on the mtls export CLI command"
+---
+
+## Description
+
+Export the root CA, issuer cert and key from Kubernetes to local files
```

Comment:
Should this mtls export say "only from K8s"? Should this not also be supported platforms? Others could be supported one day and this could be applied locally if the sentry service was running locally as described here See https://docs.dapr.io/operations/security/mtls/#self-hosted

So maybe the text should be like this?
"Export the root CA, issuer cert and key from Dapr Sentry service to local files"

and the reason here is that the Sentry service does run locally.

Reply:
right now the command does only run for Kubernetes https://github.com/dapr/cli/blob/master/pkg/kubernetes/mtls.go. 

I think we can change the help message when we start supporting other platforms.